readthedocs_build/config/config.py

```diff
@@ -30,6 +26,18 @@
 TYPE_REQUIRED = 'type-required'
 PYTHON_INVALID = 'python-invalid'
 
+DOCKER_BUILD_IMAGES = {
```

Comment:
Make this updatable from a setting as it was, so we can define a custome DOCKER_BUILD_IMAGES in our development instance.

Reply:
hmm, good call. Hadn't thought of that specific use.

Reply:
+1 on a setting, we shouldn't change that behavior. This is important for all non-prod instances.